src/github.com/getlantern/flashlight/server/server.go

```diff
@@ -222,6 +250,69 @@ func (server *Server) stopNattywad() {
 	server.waddellClient = nil
 }
 
+func (server *Server) checkForDisallowedPort(addr string) error {
+	_, portString, err := net.SplitHostPort(addr)
```

Comment:
Just checking: we never ever get just the host part and assume the port to be 80, right?